types/account.go

```diff
@@ -0,0 +1,47 @@
+package types
```

Comment:
suggestion to rename the package to `data`, just the way we have in `elrond-go` and other projects

Reply:
done.